businesscentral/image.yaml

```diff
@@ -187,6 +190,7 @@ packages:
           - python-requests
           - python-enum34
           - PyYAML
+          - git
```

Comment:
@ruromero Please add this for [RHDM](https://github.com/jboss-container-images/rhdm-7-openshift-image) as well

Reply:
Done https://github.com/jboss-container-images/rhdm-7-openshift-image/pull/62